### ISSUE TITLE
The volume.Response  volume.Request is undefined

### ISSUE THREAD

**Issue by kweisamx** [yp-engineering/rbd-docker-plugin#46]
When i build the go 
```
# go build
# _/root/rbd-docker-plugin
./driver.go:135: undefined: volume.Response
./driver.go:135: undefined: volume.Request
./driver.go:143: undefined: volume.Response
./driver.go:143: undefined: volume.Request
./driver.go:217: undefined: volume.Response
./driver.go:217: undefined: volume.Request
./driver.go:295: undefined: volume.Response
./driver.go:295: undefined: volume.Request
./driver.go:388: undefined: volume.Response
./driver.go:388: undefined: volume.Request
./driver.go:388: too many errors
[root@localhost rbd-docker-plugin]# 
```
and I check the package in volume/
there is no Response and Response function.
please check , thanks.

**Comment by kweisamx** [comment]
That's fine when we do `go get github.com/docker/go-plugins-helpers`
I suggest the Doc can add these package to install
Thanks.

**Comment by porcupie** [comment]
Should be fixed on latest master 21b9c0080675e0f770f1079e139fedb099f2e40d
